sos/plugins/kubernetes.py

```diff
@@ -22,33 +23,101 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    # Red Hat Atomic Platform and OpenShift Enterprise use the
+    # atomic-openshift-master package to provide kubernetes
+    packages = ('kubernetes', 'atomic-openshift-master')
+
+    option_list = [
+        ("all", "use --all-namespaces for all kubectl commands",
+            'fast', True),
+        ("describe", "capture descriptions of all kube resources",
+            'fast', True),
+        ("podlogs", "capture logs for pods", 'slow', False),
+    ]
+
+    def check_is_master(self):
+        return path.exists("/var/run/kubernetes/apiserver.key")
```

Comment:
This will return False on an OpenShift Enterprise / Atomic Platform master - where it should return True.
In OSE/AP the API server cert/key lives by default in `/etc/origin/master/master.server.{crt,key}` but it probably makes more sense to check for the master configuration file itself, i.e. `/etc/origin/master/master-config.yaml` (the master configuration specifies where the cert file/key is)